src/BullOak.Repositories.EventStore/AlwaysPassValidator.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace BullOak.Repositories.EventStore
+{
+    using BullOak.Repositories.Session;
+
+    public class AlwaysPassValidator<TState> : IValidateState<TState>
```

Comment:
There is already https://github.com/BullOak/BullOak/blob/master/src/BullOak.Repositories/Session/AlwaysPassValidator.cs 

Why do we need another one?

Reply:
I wanted it internal. I want this internal too tbqh, I made a mistake making it public. the idea is for consumers to either provide a normal validator OR not provide any at all (and then framework does everything).

It's just a tiny bit of duplication and it's ok to my eyes; if we add functionality to it (we shouldn't), then I'll consider making it public.

Reply:
OK, I see. Fine with me as is.